docs/DeveloperGuide.md

```diff
@@ -13,6 +13,8 @@ title: Developer Guide
    - [Storage component](#storage-component)
    - [Common classes](#common-classes)
 5. [Implementation](#implementation)
+   - [Add student feature](#add-student-feature)
```

Comment:
This should coincide with the header title that you have given, i.e. `Add student feature: add -s` . Sorry for my mistake in the previous review! This applies to `Delete student feature` too.